metabase/metabase
# Try fixing loki flakiness, fix waterfall does not render

### Description

Loki specs are flaky, usually scatter snapshots produce tiny diff around circles border. This PR loosens diff settings allowing some threshold. Also it fixes accidentally broken waterfall. It also adds `yarn test-visual:loki-report-open` command to run loki specs and open the test report page after.

### How to verify

- Run `yarn test-visual:loki-report-open` to verify it runs specs and opens the report
- Check Loki CI check status on this PR
